src/tree/registries/logInWizard/RegistryPasswordStep.ts

```diff
@@ -4,17 +4,17 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { AzureWizardPromptStep } from 'vscode-azureextensionui';
-import { ext } from '../../../../extensionVariables';
-import { IDockerHubWizardContext } from './IDockerHubWizardContext';
+import { ext } from '../../../extensionVariables';
+import { ILogInWizardContext } from './ILogInWizardContext';
 
-export class DockerHubPasswordStep extends AzureWizardPromptStep<IDockerHubWizardContext> {
-    public async prompt(context: IDockerHubWizardContext): Promise<void> {
-        const prompt: string = "Enter your password";
+export class RegistryPasswordStep extends AzureWizardPromptStep<ILogInWizardContext> {
+    public async prompt(context: ILogInWizardContext): Promise<void> {
+        const prompt = context.passwordPrompt || "Enter your password";
```

Comment:
`context.passwordPrompt` is `string | undefined`

Not intentional, but I'm not gonna change it because this repo doesn't have the `typedef` rule enabled for variables. The compiler is smart enough to know this will be a string. Stephen seems to find it annoying in some cases as well: https://github.com/microsoft/vscode-azurearmtools/pull/247#discussion_r286721245